Removed a broken link under TV

VinayMayavanshi13 opened the pull request.

VinayMayavanshi13 committed:
Removed a broken link under TV

VinayMayavanshi13 committed:
Removed Dead links

1. VPN Guides and Tutorials - Under this I removed /r/VPN wiki - As that Page has been disabled.
2. Browser Extension - Removed Universal ByPass - As it no more exists.
3. UserScripts - Removed AntiAdware and Remove fake TPB torrents - It does not exist.

VinayMayavanshi13 committed:
Removed some more broken links.

1. Windows 10 Privacy - Removed Windows Privacy Tweaker.
2. Domain Names - Removed Xip.io - As it is unavailable now.

VinayMayavanshi13 committed:
Removed dead links under Torrenting

Live Tracer, Torrage, xbit, Torrent Paradise, torrent.nz were removed.

VinayMayavanshi13 committed:
Deleted broken links

Links from Public Trackers, Seedbox Setup Tools and Guides, WebTorrents Clients,

BNX070 commented:
Why tf are you removing actual working links! if they don't work for you that doesn't mean they don't work for the rest...
First confirm and check sometimes a VPN can bypass the restrictions too.'

Mods please don't merge these changes without confirming if all the links are actually down.
soo many in the private tracker are working just fine.

VinayMayavanshi13 committed:
Removed a banned link and some broken links

Removed a banned link - Butter Project  and some other dead links.

VinayMayavanshi13 committed:
Removed Broken Links

Removed broken links under Web-based Cloud Seedboxes, Seedbox Hosting Providers.

VinayMayavanshi13 commented:
@BNX070 Thanks for letting me know. Most of the links are broken. Yes some may be accessed by VPN, In such a case, please do add that again. Sorry for the inconvenience.

VinayMayavanshi13 committed:
Solved issue #684

Removed a banned link under Miscellaneous - /r/EmbyShares

VinayMayavanshi13 committed:
Merge pull request #1 from VinayMayavanshi13/VinayMayavanshi13-patch-1

Solved issue #684
